### ISSUE TITLE
Only setting a minimum window size limit doesn't work (Tested on Linux)

### ISSUE THREAD

**Issue by CreamyCookie** [glfw/glfw#855]
**OS and version:**
Ubuntu 16.04 (64bit) — 4.4.0-36-generic GNU/Linux

**Release or commit:**
GLFW: 3.2.0 (according to GLFW.java from lwjgl)

**Problem:**
I'm using LibGDX, which uses LWJGL 3 (among others), which uses GLFW. When I call

```java
cfg.setWindowSizeLimits(640, 480, 999999999, 999999999);
```

it works. When I use

```java
cfg.setWindowSizeLimits(640, 480, -1, -1);
```

it doesn't seem to work. I can make the window smaller than 640 x 480.

Note that `-1` **[gets transformed](https://github.com/libgdx/libgdx/blob/master/backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Application.java#L415)** to `GLFW_DONT_CARE`.

As I don't know much about GLFW, I can only provide an SSCCE using LibGDX:

```java
import com.badlogic.gdx.ApplicationListener;
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;

public class DesktopLauncher {

    public static void main(String[] arg) {
        Lwjgl3ApplicationConfiguration cfg = new Lwjgl3ApplicationConfiguration();

        cfg.useOpenGL3(true, 3, 3);
        cfg.useVsync(false);
        cfg.setWindowSizeLimits(640, 480, -1, -1);
        cfg.setResizable(true);


        new Lwjgl3Application(new ApplicationListener() {
            @Override
            public void create() {
                
            }

            @Override
            public void resize(int width, int height) {

            }

            @Override
            public void render() {

            }

            @Override
            public void pause() {

            }

            @Override
            public void resume() {

            }

            @Override
            public void dispose() {

            }
        }, cfg);
    }
}
```

**Comment by tombsar** [comment]
I believe this was fixed in glfw 3.2.1 (see #805).

**Comment by CreamyCookie** [comment]
Oh nice. (Didn't find this when I was searching.) Thanks! Closing.
